tools/src/main/java/org/apache/rocketmq/tools/command/cluster/ClusterListSubCommand.java

```diff
@@ -113,7 +113,7 @@ private void printClusterMoreStats(final DefaultMQAdminExt defaultMQAdminExt) th
         while (itCluster.hasNext()) {
             Map.Entry<String, Set<String>> next = itCluster.next();
             String clusterName = next.getKey();
-            TreeSet<String> brokerNameSet = new TreeSet<String>();
```

Comment:
@dongeforever  

Do we still need to make rocketmq-tools be compatible  with jdk1.6? If yes, we can not do the changes in rocketmq-tools module.